Packet.js

```diff
@@ -1,14 +1,48 @@
+const buffer2struct = require('./ESP3/buffer2struct.js')
+const packetPrototypes = {
+  0: require('./ESP3/ESP3Packet.js'),
+  1: require('./ESP3/PacketTypes/RadioERP1.js'),
+  2: require('./ESP3/PacketTypes/Response'),
+  3: require('./ESP3/PacketTypes/RadioSubTel'),
+  4: require('./ESP3/PacketTypes/Event'),
+  5: require('./ESP3/PacketTypes/CommonCommand'),
+  6: require('./ESP3/PacketTypes/SmartAckCommand'),
+  7: require('./ESP3/PacketTypes/RemoteManCommand'),
+  9: require('./ESP3/PacketTypes/RadioMessage'),
+  0xa: require('./ESP3/PacketTypes/RadioERP2'),
```

Comment:
Why HEX?

Reply:
no specific reason... i just fellt like it... i'm happy to use dec or oct or bin or whatever base seems to fit... maybe because telegram string representations are mostly given in hex, so it's more readable that way...